src/ResourceManager/EventHub/Commands.EventHub/help/Get-AzureRmEventHubDRConfigurations.md

```diff
@@ -0,0 +1,104 @@
+---
+external help file: Microsoft.Azure.Commands.EventHub.dll-Help.xml
+Module Name: AzureRM.EventHub
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmEventHubDRConfigurations
+
+## SYNOPSIS
+Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## SYNTAX
+
+```
+Get-AzureRmEventHubDRConfigurations [-ResourceGroupName] <String> [-Namespace] <String> [[-Name] <String>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmEventHubDRConfigurations** Retrieves Alias(Disaster Recovery configuration) for primary or secondary namespace
+
+## EXAMPLES
+
+### Example 1
```

Comment:
@v-Ajnava please make sure the examples in your markdown files have output for the cmdlet call

Reply:
Resolved.